score-jClouds/src/main/java/io/cloudslang/content/jclouds/services/helpers/Utils.java

```diff
@@ -0,0 +1,198 @@
+package io.cloudslang.content.jclouds.services.helpers;
+
+import com.google.common.collect.Multimap;
+import io.cloudslang.content.jclouds.entities.*;
+import io.cloudslang.content.jclouds.entities.constants.Constants;
+import io.cloudslang.content.jclouds.entities.inputs.ImageInputs;
+import io.cloudslang.content.jclouds.entities.inputs.InstanceInputs;
+import io.cloudslang.content.jclouds.utils.InputsUtil;
+import org.apache.commons.lang3.StringUtils;
+import org.jclouds.ContextBuilder;
+
+import java.io.Closeable;
+
+/**
+ * Created by Mihai Tusa.
+ * 6/15/2016.
+ */
+public class Utils {
+    public <T extends Closeable> T getApi(ContextBuilder contextBuilder, Class<T> valueType) {
+        return contextBuilder.buildApi(valueType);
+    }
+
+    void updateInstanceFiltersMap(InstanceInputs instanceInputs, Multimap<String, String> filtersMap, String delimiter) {
+        setInstanceRelevantFilters(instanceInputs, filtersMap);
+        setTagFilters(instanceInputs, filtersMap, delimiter);
+
+        updateFiltersMapEntry(filtersMap, CommonFilters.KERNEL_ID.getValue(), instanceInputs.getCustomInputs().getKernelId());
```

Comment:
the CommonFilters enum should be renamed to CommonFilter(same for the other enums)